tests/e2e/bin/setup.sh

```diff
@@ -148,12 +148,32 @@ rm -rf $E2E_ROOT/woocommerce-subscriptions.zip
 
 redirect_output cli wp plugin activate woocommerce-subscriptions
 
+step "Installing Woo Pre-Orders"
+echo " - Fetching latest version"
+LATEST_RELEASE_ASSET_ID=$(curl -sH "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/woocommerce/woocommerce-pre-orders/releases/latest | jq -r '.assets[0].id')
+
+redirect_output curl -sLJ \
+	-H "Authorization: token $GITHUB_TOKEN" \
+	-H "Accept: application/octet-stream" \
+	--output $E2E_ROOT/woocommerce-pre-orders.zip \
+	https://api.github.com/repos/woocommerce/woocommerce-pre-orders/releases/assets/"$LATEST_RELEASE_ASSET_ID"
```

Comment:
Since this is an open-source plugin that anyone can contribute to, only people with access to this repository would be able to run the E2E tests. Wondering if we should not run pre-order tests by default.

Reply:
Or perhaps skip pre-order tests if the plugin isn’t installed successfully?

Reply:
Good call. Addressed in [d838091](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3932/commits/d83809168818aa2d304437e0e488372d4fd0be05).